composer.json

```diff
@@ -8,8 +8,7 @@
         }
```

Comment:
I noticed within our Assets/AssetType\Image class we have some references in regards to this class.

use League\Glide\Urls\UrlBuilderFactory;

// methods 
- dynamic
- getBuilder